templates/master/00-master/azure/files/opt-libexec-openshift-azure-routes-sh.yaml

```diff
@@ -131,7 +131,11 @@ contents:
     remove_stale_routes() {
         ## find extra ovn routes
         local ovnkContainerID=($(crictl ps --name ovnkube-controller | awk '{ print $1 }' | tail -n+2))
-        echo ${ovnkContainerID}
+        if [ -z "${ovnkContainerID:-}" ]; then
```

Comment:
then you can remove the :- here, because the variable wont be unset after you remove the subshell stuff

Reply:
yep done. will start a SDN cluster and ensure its good.

Reply:
```
Sep 21 15:58:11 ci-ln-jy7jqk2-1d09d-crtst-master-1 openshift-azure-routes[48934]: removing stale vip 10.0.0.4 for local clients
Sep 21 15:58:11 ci-ln-jy7jqk2-1d09d-crtst-master-1 openshift-azure-routes[48934]: Plugin is SDN, nothing to do.. exiting
Sep 21 15:58:12 ci-ln-jy7jqk2-1d09d-crtst-master-1 openshift-azure-routes[48934]: Plugin is SDN, nothing to do.. exiting
Sep 21 15:58:12 ci-ln-jy7jqk2-1d09d-crtst-master-1 openshift-azure-routes[48934]: done applying vip rules
```
seems to work